Dockerfile

```diff
@@ -731,6 +731,7 @@ COPY --chmod=0644 hack/containerd.toml /rootfs/etc/containerd/config.toml
 COPY --chmod=0644 hack/cri-containerd.toml /rootfs/etc/cri/containerd.toml
 COPY --chmod=0644 hack/cri-plugin.part /rootfs/etc/cri/conf.d/00-base.part
 COPY --chmod=0644 hack/udevd/80-net-name-slot.rules /rootfs/usr/lib/udev/rules.d/
+COPY --chmod=0644 hack/udevd/xx-selinux.rules /rootfs/usr/lib/udev/rules.d/
```

Comment:
In what place in order should we place it? Perhaps 99?

Reply:
Set 90 now

Reply:
90 feels good, maybe we should also do a validation that the user doesn't override these?

Reply:
Well, user is the Talos admin, not the k8s admin who we protect the system from. Perhaps should be okay

Reply:
I guess it's fine, on second thoughts I believe we allow overwriting existing udev rules, so should be okay